tools/performance/engine-benchmarks/bench_tool/bench_results.py

```diff
@@ -164,14 +166,22 @@ async def get_bench_report(bench_run: JobRun, temp_dir: str, remote_cache: Remot
     zip_file = zipfile.ZipFile(zip_file_name, "r")
     zip_file.extractall(extracted_dirname)
     bench_report_xml = path.join(extracted_dirname, "bench-report.xml")
-    assert path.exists(bench_report_xml)
+    bench_report_json = path.join(extracted_dirname, "bench-report.json")
+    if path.exists(bench_report_xml):
```

Comment:
Keep backward compatibility - if there is an "old" `bench-report.xml` artifact, process it. This means that all the older benchmark runs still can be processed.